File: April_2022/02_04_2022/plants.py
p=int(input())
s=int(input())
x=1
days=int(input())
w=0
while x<=days:
    if (x+2)<=days:
        p+=s
        w+=(p-s)
    else:
        w+=p
    x+=2
print(p)
print(w)

File: April_2022/02_04_2022/plants.cpp
#include <iostream>

using namespace std;

int main(){

    int p,s,days;
    cin>>p>>s>>days;

    int x = 1;
    int w = 0;
    
    while(x<=days){
        if((x+2)<=days){
            p += s;
            w += (p-s);
        }
        else{
            w += p;
        }
        x += 2;
    }

    cout<<p<<endl;
    cout<<w<<endl;

    return 0;
}